soroban-sdk/src/testutils.rs

```diff
@@ -416,3 +419,98 @@ pub trait Deployer {
     /// the provided address, or if the instance/code has expired.
     fn get_contract_code_ttl(&self, contract: &crate::Address) -> u32;
 }
+
+pub use xdr::AccountFlags as IssuerAccountFlags;
+
+pub struct StellarAssetIssuer {
+    pub env: Env,
+    pub account_id: xdr::AccountId,
```

Comment:
I think these fields should be private, or if we need access within the crate, `pub(crate)`. That's most consistent with other types like the client, and if all the fields are public we're committing to allowing people to construct it from the primitives in the future and it becomes a breaking change to add either a public or private field.

Reply:
Good catch. Updated to private and exposed any relevant fields through getters.